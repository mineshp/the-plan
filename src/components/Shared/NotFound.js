import React from 'react';
import { Container, Header } from 'semantic-ui-react';

const NotFound = () => (
    <div>
        <Container text>
            <Header as="h2">404 Page Not Found!</Header>
            <p>Oops it looks like you have ventured off-course.</p>
        </Container>
    </div>
);

export default NotFound;
