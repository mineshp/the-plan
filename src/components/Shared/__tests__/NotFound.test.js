import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../NotFound';

describe('NotFound 404 component', () => {
    it('renders correctly', () => {
        const NotFoundComponent = shallow(
            <NotFound />
        );

        expect(NotFoundComponent).toMatchSnapshot();
    });
});
