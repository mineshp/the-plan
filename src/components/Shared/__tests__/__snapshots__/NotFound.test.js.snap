// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound 404 component renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    <Container
        text={true}
    >
        <Header
            as="h2"
        >
            404 Page Not Found!
        </Header>
        <p>
            Oops it looks like you have ventured off-course.
        </p>
    </Container>
</div>,
  "nodes": Array [
    <div>
      <Container
            text={true}
      >
            <Header
                  as="h2"
            >
                  404 Page Not Found!
            </Header>
            <p>
                  Oops it looks like you have ventured off-course.
            </p>
      </Container>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <NotFound />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div>
          <Container
                    text={true}
          >
                    <Header
                              as="h2"
                    >
                              404 Page Not Found!
                    </Header>
                    <p>
                              Oops it looks like you have ventured off-course.
                    </p>
          </Container>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div>
          <Container
                    text={true}
          >
                    <Header
                              as="h2"
                    >
                              404 Page Not Found!
                    </Header>
                    <p>
                              Oops it looks like you have ventured off-course.
                    </p>
          </Container>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <NotFound />,
}
`;
